bedrock/firefox/templates/firefox/new/desktop/download.html

```diff
@@ -63,6 +73,11 @@ <h2 class="c-sub-navigation-title">{{ ftl('sub-navigation-firefox-for-desktop',
 {% endblock %}
 
 {% block content %}
+
+  {% if show_yandex %}
+    {% include 'firefox/new/desktop/includes/yandex.html' %}
+  {% endif %}
```

Comment:
Would changing this `endif` to an `else` help make the markup a bit cleaner? (avoid two `<main>` elements, `<h1>` etc in the source code?

Reply:
Oh right, we can’t do this as it’s all client side. Never mind :/

Reply:
Because the geoLocation swap needs to be done client side, both of these chuncks need to go on the page.

I've re-jigged it not to use two `<main>` elements. Only one `<h1>` will be visible at a time. I don't _think_ it's worth removing the hidden one from the DOM since the search engines will either pick up on it before the page renders or treat it as hidden after it renders.